script/linux-build-docker

```diff
@@ -67,6 +67,7 @@ if [[ "${PLATFORM}" == "flex" ]]; then
   cat "$FLEX_DIR/nginx_proxy/Dockerfile"
 fi
 
+chmod +w "${DOCKERFILE_PATH}/endpoints-runtime.deb"
```

Comment:
won't this fail if `${DOCKERFILE_PATH}/endpoints-runtime.deb` doesn't exist?
since we're copying below, simply delete this would suffice.

Reply:
Make sense.